src/Plugin/migrate/process/NameField.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\stanford_migrate\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Parse a string from common name formats.
+ *
+ * Examples:
+ *
+ * @code
+ * process:
+ *   plugin: name_field
+ *   source: some_text_field
+ * @endcode
+ *
+ * @MigrateProcessPlugin(
+ *   id = "name_field"
+ * )
+ */
+class NameField extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    $parser = new \FullNameParser();
```

Comment:
I think you need a `use` statement for the sake of clarity.

Reply:
come to think of it, there's a static method in that class, so you could just have a `use` statement and do `FullNameParser::parse($value)`, but six of one, half-dozen of the other.

Reply:
Ooh I like the static method. Good find